### DISCUSSION TITLE
how to embed video use reactplayer?

### DISCUSSION THREAD

**Discussion by abrahamcahyoadi** [facebook/docusaurus#8082] [Q&A] [answered]
Hello all can you help me how to embed video use reactplayer?

step use reactplayer:

1.open terminal
`npm i react-player`
`npm start`

2.Create new docs Laravel-9-Pusher.md 
3.add code embed video <ReactPlayer playing controls src='/1.mp4' />
![why](https://user-images.githubusercontent.com/39833970/189536194-3f893a26-aa83-4bbf-96bc-2ef53c88301b.PNG)

how to import Reactplayer in docusaurus?

**Comment by kaitoukid-1412** [comment, marked as answer]
Hi,

You need change name file `Laravel-9-Pusher.md` to `Laravel-9-Pusher.mdx`

Then:

```js
import ReactPlayer from "react-player/youtube";

export default function Function() {
  return <ReactPlayer url="https://www.youtube.com/watch?v=ysz5S6PUM-U" />
}
```

![image](https://user-images.githubusercontent.com/15698245/189574586-5f982c59-787e-4b40-90d0-364a333bd4b9.png)

**Reply by Josh-Cena** [reply]
@Shenlong616 FYI, `.md` and `.mdx` are exactly the same.

**Reply by abrahamcahyoadi** [reply]
Thanks working @Shenlong616 and @Josh-Cena

i use format `.md`
`import ReactPlayer from "react-player";
<ReactPlayer playing controls url='/1.mp4' />`
